### PR TITLE
Regenerating binstub for rspec-core

### PR DESCRIPTION
**Pull request by adomokos**
This commit fixes this warning:

old_binstub_for_rspec

And this is after:

added_new_rspec-core_binstub

### CODE REVIEW

**Comment by bf4** [comment]
:100:

File: bin/rspec [hunk 1]
```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
-end
```
Comment (bf4) [inline bin/rspec]: full disclosure: you're removing spring here.  Just calling it out. Spring causes so many weird errors.

**Comment by bf4** [comment]
?

**Comment by adomokos** [comment]
Nothing happened for 8-9 months. I figured it's not worthy for merging.

**Pull request closed without merging**